apps/src/code-studio/pd/form_components_func/FormController.jsx

```diff
@@ -60,22 +61,23 @@ InvalidPagesSummary.propTypes = {
 const FormController = props => {
   const {
     pageComponents,
-    requiredFields = [],
+    requiredFields,
     apiEndpoint,
-    applicationId = undefined,
-    allowPartialSaving = false,
+    applicationId,
+    allowPartialSaving,
+    autoComputedFields,
     options,
-    getInitialData = () => ({}),
-    onInitialize = () => {},
-    onSetPage = () => {},
-    onSuccessfulSubmit = () => {},
-    onSuccessfulSave = () => {},
-    serializeAdditionalData = () => ({}),
-    sessionStorageKey = null,
-    submitButtonText = defaultSubmitButtonText,
+    getInitialData,
+    onInitialize,
+    onSetPage,
+    onSuccessfulSubmit,
+    onSuccessfulSave,
+    serializeAdditionalData,
+    sessionStorageKey,
+    submitButtonText,
     getPageProps: getAdditionalPageProps = () => ({}),
```

Comment:
I tried to move this too, but it caused an error. See commit history to see it

Reply:
huh, not sure why that was happening, but shouldn't hurt